<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>952</width>
    <height>522</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>85</red>
        <green>170</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>85</red>
        <green>170</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>85</red>
        <green>170</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>85</red>
        <green>170</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="font">
   <font>
    <family>Papyrus</family>
    <pointsize>10</pointsize>
    <italic>true</italic>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resources.qrc">
    <normaloff>:/Cubana_Airlines_logo.svg.png</normaloff>:/Cubana_Airlines_logo.svg.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>150</y>
      <width>931</width>
      <height>351</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="icon">
      <iconset resource="Resources.qrc">
       <normaloff>:/user-home.ico</normaloff>:/user-home.ico</iconset>
     </attribute>
     <attribute name="title">
      <string>Home</string>
     </attribute>
     <widget class="QCalendarWidget" name="calendarWidget">
      <property name="geometry">
       <rect>
        <x>12</x>
        <y>11</y>
        <width>256</width>
        <height>161</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="notifications">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>210</y>
        <width>751</width>
        <height>91</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>180</y>
        <width>321</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; color:#5555ff;&quot;&gt;Noticias&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdNumber">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>10</y>
        <width>471</width>
        <height>131</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="digitCount">
       <number>12</number>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="icon">
      <iconset resource="Resources.qrc">
       <normaloff>:/2081-44578.png</normaloff>:/2081-44578.png</iconset>
     </attribute>
     <attribute name="title">
      <string>Vuelos</string>
     </attribute>
     <widget class="QLineEdit" name="buscar">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>10</y>
        <width>351</width>
        <height>27</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Especifique el destino al que desea buscar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="search">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>10</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="palette">
       <palette>
        <active>
         <colorrole role="Button">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="Button">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="Button">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="Resources.qrc">
        <normaloff>:/Search-dusk.png</normaloff>:/Search-dusk.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>881</width>
        <height>221</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="icon">
      <iconset resource="Resources.qrc">
       <normaloff>:/matrix2 crasybrowser.png</normaloff>:/matrix2 crasybrowser.png</iconset>
     </attribute>
     <attribute name="title">
      <string>Reservaciones</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>70</y>
        <width>891</width>
        <height>231</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>3</number>
      </property>
      <property name="iconSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
      <widget class="QWidget" name="tab_4">
       <attribute name="icon">
        <iconset resource="Resources.qrc">
         <normaloff>:/Msn.ico</normaloff>:/Msn.ico</iconset>
       </attribute>
       <attribute name="title">
        <string/>
       </attribute>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>391</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Primero sus datos personales. Aségurese de que son correctos.</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="name">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>351</width>
          <height>20</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>Nombre</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lname">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>50</y>
          <width>351</width>
          <height>20</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>Apellidos</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="pssp">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>70</y>
          <width>351</width>
          <height>20</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>No. de Pasaporte</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="icon">
        <iconset resource="Resources.qrc">
         <normaloff>:/LG chocolate.png</normaloff>:/LG chocolate.png</iconset>
       </attribute>
       <attribute name="title">
        <string/>
       </attribute>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>741</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Ahora déjenos un teléfono para localizarlo. Aségurese de brindar al menos uno. O en caso extremo notificar su ausencia.</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="phone1">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>40</y>
          <width>221</width>
          <height>20</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>Introduzca No. de teléfono</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="nphone">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>50</y>
          <width>181</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>No poseo teléfono</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_6">
       <attribute name="icon">
        <iconset resource="Resources.qrc">
         <normaloff>:/New Mail.png</normaloff>:/New Mail.png</iconset>
       </attribute>
       <attribute name="title">
        <string/>
       </attribute>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>741</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Ahora déjenos un email para localizarlo. Aségurese de brindar al menos uno. O en caso extremo notificar su ausencia.</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="email1">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>50</y>
          <width>231</width>
          <height>20</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>Introduzca email</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_2">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>60</y>
          <width>171</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>No poseo email</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_7">
       <attribute name="icon">
        <iconset resource="Resources.qrc">
         <normaloff>:/Internet Shortcut (3).png</normaloff>:/Internet Shortcut (3).png</iconset>
       </attribute>
       <attribute name="title">
        <string/>
       </attribute>
       <widget class="QComboBox" name="dest">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>40</y>
          <width>241</width>
          <height>22</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>Seleccione su destino</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Spain</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Sumatra</string>
         </property>
        </item>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>80</y>
          <width>241</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Especifique la  fecha en que va a viajar.</string>
        </property>
       </widget>
       <widget class="QDateEdit" name="dateEdit">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>110</y>
          <width>161</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="Line" name="line">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>10</y>
          <width>20</width>
          <height>171</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>10</y>
          <width>271</width>
          <height>151</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:600; color:#5555ff;&quot;&gt;Gracias por&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:600; color:#5555ff;&quot;&gt; Preferirnos.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="Line" name="line_2">
        <property name="geometry">
         <rect>
          <x>590</x>
          <y>10</y>
          <width>20</width>
          <height>171</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_8">
        <property name="geometry">
         <rect>
          <x>620</x>
          <y>10</y>
          <width>211</width>
          <height>20</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>Escoja un vuelo y pulse reservar</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>640</x>
          <y>80</y>
          <width>201</width>
          <height>81</height>
         </rect>
        </property>
        <property name="text">
         <string>Resrvar</string>
        </property>
        <property name="icon">
         <iconset resource="Resources.qrc">
          <normaloff>:/matrix2 crasybrowser.png</normaloff>:/matrix2 crasybrowser.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>681</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; color:#5555ff;&quot;&gt;Siga los pasos y asegurese de brindar toda la información requerida.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QCommandLinkButton" name="visitenos">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Visítenos</string>
    </property>
    <property name="icon">
     <iconset resource="Resources.qrc">
      <normaloff>:/Internet.png</normaloff>:/Internet.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="logo">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>291</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/Cubana_Airlines_logo.svg.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>952</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuVuelos">
    <property name="title">
     <string>Vuelos</string>
    </property>
    <addaction name="actionBuscar"/>
   </widget>
   <widget class="QMenu" name="menuReservaciones">
    <property name="title">
     <string>Reservaciones</string>
    </property>
    <addaction name="actionReservar"/>
   </widget>
   <widget class="QMenu" name="menuOpciones">
    <property name="title">
     <string>Opciones</string>
    </property>
    <addaction name="actionSeleccionar_Idioma"/>
    <addaction name="actionConexi_n"/>
    <addaction name="separator"/>
    <addaction name="actionSalir"/>
   </widget>
   <addaction name="menuVuelos"/>
   <addaction name="menuReservaciones"/>
   <addaction name="menuOpciones"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Highlight">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Highlight">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Highlight">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>51</red>
         <green>153</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="iconSize">
    <size>
     <width>28</width>
     <height>28</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionBuscar"/>
   <addaction name="actionReservar"/>
   <addaction name="separator"/>
   <addaction name="actionSeleccionar_Idioma"/>
   <addaction name="actionConexi_n"/>
   <addaction name="separator"/>
   <addaction name="actionSalir"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionBuscar">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Search-dusk.png</normaloff>:/Search-dusk.png</iconset>
   </property>
   <property name="text">
    <string>Buscar</string>
   </property>
  </action>
  <action name="actionReservar">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/matrix2 crasybrowser.png</normaloff>:/matrix2 crasybrowser.png</iconset>
   </property>
   <property name="text">
    <string>Reservar!</string>
   </property>
  </action>
  <action name="actionSeleccionar_Idioma">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/UK.ico</normaloff>:/UK.ico</iconset>
   </property>
   <property name="text">
    <string>Seleccionar Idioma</string>
   </property>
  </action>
  <action name="actionConexi_n">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/LAN.png</normaloff>:/LAN.png</iconset>
   </property>
   <property name="text">
    <string>Conexión</string>
   </property>
  </action>
  <action name="actionSalir">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/4837-LloydBanks-Areternnselec.ico</normaloff>:/4837-LloydBanks-Areternnselec.ico</iconset>
   </property>
   <property name="text">
    <string>Salir</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
